### ISSUE TITLE
Infra tooling automation

### ISSUE THREAD

**Issue by beatrizmcouto** [RedHatProductSecurity/trestle-bot#296]

**Comment by jpower432** [comment]
@beatrizmcouto Does this duplicate #303?

**Comment by beatrizmcouto** [comment]
@jpower432 yes it does, thank you for spotting it. It was on my to do list and I didn't realize it was already here. I will delete 303

**Comment by marcusburghardt** [comment]
Closing as part of repository archival.

ComplyScribe is being archived because [complyctl](https://github.com/complytime/complyctl) has been redesigned to use [Gemara](https://github.com/complytime/gemara) as its core engine. complyctl can still generate OSCAL as output but no longer requires OSCAL as input, making complyscribe's content transformation pipeline obsolete.

For the current compliance tooling, see [complyctl](https://github.com/complytime/complyctl).
